stockpile/roles/httpd/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+  # Allow for capturing of the httpd configuration
+  # TODO: Remove hardcoding of indice values
+  - name: Get keepalive settings
+    shell:
+       grep "KeepAlive" /etc/conf/httpd.conf | cut -d ' ' -f 2
+    register: keep_alive
+    ignore_errors: true
+  - shell:
+       grep "MaxKeepAliveRequests" /etc/conf/httpd.conf | cut -d ' ' -f 2
+    register: max_keepalive_requests
```

Comment:
max_keepalive_requests is already used for https://github.com/redhat-performance/stockpile/pull/18/files#diff-96c5490205a988f090baee83580a3545R24 not sure how different these two are, but in any case I'd suggest not to use a generic naming convention, instead add a prefix as it'll make it easier to extract facts needed based on requirements and won't cause any conflict as well